### PR TITLE
Added improvmx verification txt, remove email from discord and github pages json

### PR DESCRIPTION
**Pull request by ghost**
- [x] The file is in the `domains` folder and is in the JSON format.
- [x] The file's name is all lowercased and alphanumeric. 
- [ ] You have completed your website. 
- [x] The website is reachable.  
- [x] You're not using Vercel.  
- [x] The CNAME record doesn't contain `https://` or `/`.  
- [x] There is sufficient information at the `owner` field.  

Added a txt domain for improvmx to verify my domain as my old email was unreachable (deleted) 
image

Removed email from _discord.axviii3.json and _github-pages-challenge-axviii3.axviii3.json and added discord to both

### CODE REVIEW

File: domains/_discord.axviii3.json [hunk 1]
```diff
@@ -1,7 +1,7 @@
 {
   "owner": {
     "username": "AXVIII3",
-    "email": "anirbanrc2286@gmail.com"
+    "discord": "axviii3"
```
Comment (CuteDog5695) [inline domains/_discord.axviii3.json:4]:
```suggestion
    "discord": "axviii3",
    "email": ""
```

File: domains/_github-pages-challenge-axviii3.axviii3.json [hunk 2]
```diff
@@ -1,7 +1,7 @@
 {
   "owner": {
     "username": "AXVIII3",
-    "email": "anirbanrc2286@gmail.com"
+    "discord": "axviii3"
```
Comment (CuteDog5695) [inline domains/_github-pages-challenge-axviii3.axviii3.json:4]:
```suggestion
    "discord": "axviii3",
    "email": ""
```

File: domains/axviii3.json [hunk 3]
```diff
@@ -1,7 +1,6 @@
 {
   "owner": {
     "username": "AXVIII3",
-    "email": "",
     "discord": "axviii3"
```
Comment (CuteDog5695) [inline domains/axviii3.json:4]:
```suggestion
    "discord": "axviii3",
    "email": ""
```

**Review by iostpa — DISMISSED** [review]

**Comment by axviii3** [comment]
@CuteDog5695 
Updated with changes... Thanks...

If you dont mind, why is the empty email field required?

**Review by MaskDuck — APPROVED** [review]

**Comment by axviii3** [comment]
> so our CI won't complain

Cool thanks!

**Pull request merged**